dwellingly/src/views/addProperty.js

```diff
@@ -0,0 +1,162 @@
+import React, { Component } from 'react';
+import { Form, Field, Formik } from 'formik';
+import * as Yup from 'yup';
+import { UserContext } from '../App';
+import * as axios from 'axios';
+
+const validationSchema = Yup.object().shape({
+    name: Yup.string()
+        .min(5, "*Name must have at least 2 characters")
+        .max(50, "*Names can't be longer than 100 characters")
```

Comment:
Getting mixed signals here 😄 Sounds like the min should be 2 and the max should be 100